.github/workflows/helm-release.yaml

```diff
@@ -78,6 +78,8 @@ jobs:
         run: |
           mkdir -p /tmp/chart
           cd charts
+          # Update the prefect version tag in values.yaml
+          sed -i "s/prefectTag:.*$/prefectTag: $PREFECT_VERSION-python3.11/g" prefect-worker/values.yaml
```

Comment:
Does the version include the kubernetes tag? I'm assuming yes

Reply:
oh - it doesnt!

Reply:
well actually im not sure let me check